p2p/pex_reactor.go

```diff
@@ -47,18 +52,22 @@ type PEXReactor struct {
 	book              *AddrBook
 	ensurePeersPeriod time.Duration
 
+	// Seed/Crawler Mode
+	runningSeedCrawler bool
+
 	// tracks message count by peer, so we can prevent abuse
 	msgCountByPeer    *cmn.CMap
 	maxMsgCountByPeer uint16
 }
 
 // NewPEXReactor creates new PEX reactor.
-func NewPEXReactor(b *AddrBook) *PEXReactor {
+func NewPEXReactor(b *AddrBook, sm bool) *PEXReactor {
```

Comment:
`seedMode`. how otherwise one should guess what `sm bool` is? there is no comment

Reply:
also. should this be an option for pex reactor? or we should create `PexReactorSeedMode` given how different are normal vs seed behaviours

Reply:
I have changed the parameter name, yet regarding the method to pass the option, I struggled with this as well. I'll gladly change it to whichever method everyone agrees is best.

Reply:
@ebuchman what do you think?